Platform/servicecontrol/monitoring-instance/.diagnostics/Particular.Monitoring-configuration.txt

```diff
@@ -0,0 +1 @@
ages":{"CustomConventionUsed":false,"NumberOfMessagesFoundAtStartup":5,"Messages":["NServiceBus.ScheduledTask","ServiceControl.Monitoring.Messaging.TaggedLongValueOccurrence","NServiceBus.Metrics.MetricReport","ServiceControl.Monitoring.Messaging.RawMessage","NServiceBus.Metrics.EndpointMetadataReport"]},"NServiceBus.Autofac":{"UsingExistingLifetimeScope":true},"NServiceBus.Transport.SqlServer.CircuitBreaker":{"TimeToWaitBeforeTriggering":{"Ticks":300000000,"Days":0,"Hours":0,"Milliseconds":0,"Minutes":0,"Seconds":30,"TotalDays":0.00034722222222222218,"TotalHours":0.0083333333333333332,"TotalMilliseconds":30000,"TotalMinutes":0.5,"TotalSeconds":30}},"NServiceBus.Transport.SqlServer.DelayedDelivery":{"Native":false},"NServiceBus.Transport.SqlServer.ExpiredMessagesPurger":{"FeatureEnabled":false,"BatchSize":null},"NServiceBus.Transport.SqlServer.TimeoutManager":{"FeatureEnabled":true},"NServiceBus.Transport.SqlServer.Transactions":{"TransactionMode":1,"IsolationLevel":2,"Timeout":{"Ticks":600000000,"Days":0,"Hours":0,"Milliseconds":0,"Minutes":1,"Seconds":0,"TotalDays":0.00069444444444444436,"TotalHours":0.016666666666666666,"TotalMilliseconds":60000,"TotalMinutes":1,"TotalSeconds":60}},"Persistence":{"Sagas":{"Type":"NServiceBus.InMemoryPersistence","Version":"7.1.4"},"Timeouts":{"Type":"NServiceBus.InMemoryPersistence","Version":"7.1.4"},"Subscriptions":{"Type":"NServiceBus.InMemoryPersistence","Version":"7.1.4"},"Outbox":{"Type":"NServiceBus.InMemoryPersistence","Version":"7.1.4"},"GatewayDeduplication":{"Type":"NServiceBus.InMemoryPersistence","Version":"7.1.4"}},"Receiving":{"LocalAddress":"Particular.Monitoring@[dbo]@[ParticularMonitoringDemo]","InstanceSpecificQueue":null,"LogicalAddress":{"Qualifier":null,"EndpointInstance":{"Endpoint":"Particular.Monitoring","Discriminator":null,"Properties":{"Schema":"dbo","Catalog":"ParticularMonitoringDemo"}}},"PurgeOnStartup":false,"QueueNameBase":"Particular.Monitoring","TransactionMode":"ReceiveOnly","MaxConcurrency":4,"Satellites":[{"Name":"Timeout Message Processor","ReceiveAddress":"Particular.Monitoring.Timeouts@[dbo]@[ParticularMonitoringDemo]","TransactionMode":"ReceiveOnly","MaxConcurrency":4},{"Name":"Timeout Dispatcher Processor","ReceiveAddress":"Particular.Monitoring.TimeoutsDispatcher@[dbo]@[ParticularMonitoringDemo]","TransactionMode":"ReceiveOnly","MaxConcurrency":4}]},"Recoverability":{"ImmediateRetries":5,"DelayedRetries":3,"DelayedRetriesTimeIncrease":"0:00:10","ErrorQueue":"error","UnrecoverableExceptions":["NServiceBus.MessageDeserializationException"]},"Serialization":{"DefaultSerializer":{"Type":"NServiceBus.NewtonsoftSerializer","Version":"2.1.0","ContentType":"application/json"},"AdditionalDeserializers":[{"Type":"ServiceControl.Monitoring.Messaging.TaggedLongValueWriterOccurrenceSerializerDefinition","Version":"2.0.0","ContentType":"TaggedLongValueWriterOccurrence"}]},"Transport":{"Type":"ServiceControl.Transports.SQLServer.ServiceControlSQLServerTransport","Version":"0.0.0"}}
```

Comment:
This folder should be ignored I think. We don't want to check in the diagnostics

Reply:
pushed a fix